<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caseimageform</class>
 <widget class="QWidget" name="caseimageform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>531</width>
     <height>621</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>就诊号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="caseimage_id">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>影像上传日期：</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>人工诊断状态：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="diagnostic_state">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AI 诊 断 状 态：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="aidiagnostic_state">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>人工诊断结果：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="results">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>AI诊断结果：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="airesults">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>诊断报告：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="report">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>审核日期：</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="audit_date">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>审核状态：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="audit_status">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>590</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>姓    名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>54</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>411</width>
     <height>621</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>病例影像：</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="imagepath">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="image1">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>10</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>image1</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>人工标注影像：</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>391</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="ai_imagepath">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ai_image">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>备注信息：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadimage">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>上传影像文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savefile">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>上传标注文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>人工标注</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>智能诊断</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>分析报告</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>951</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>病例搜索：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>就诊号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>36</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>就诊日期：</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>36</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>       姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>快速搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>36</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主治医师：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
